doc/source/cluster/slurm.rst

```diff
@@ -141,6 +147,40 @@ Finally, you can invoke your Python script:
    :language: bash
    :start-after: __doc_script_start__
 
+.. _slurm-network-ray:
+
+SLURM networking caveats
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+There are two important networking aspects to keep in mind when working with
+SLURM and Ray:
+
+1. Ports binding.
+2. IP binding.
+
+One common use of a SLURM cluster is to have multiple users running concurrent
+jobs on the same infrastructure. This can easily conflict with Ray due to the
+way the head node communicates with its workers.
+
+Considering 2 users, if they both schedule a SLURM job using Ray
+at the same time, they are both creating a head node. In the backend, Ray will
+assign some internal ports to a few services. The issue is that as soon as the
+first head node is created, it will bind some ports and prevent them to be
+used by another head node. To prevent any conflicts, users have to manually
+specify non overlapping ranges of ports. The following ports are to be
+adjusted::
+
+    --node-manager-port
+    --object-manager-port
+    --port
+    --gcs-server-port
+    --min-worker-port
+    --max-worker-port
+
+As for the IP binding, on some cluster architecture the network interfaces
+do not allow to use external IPs between nodes. Instead, there are internal
+network interfaces (`eth0`, `eth1`, etc.). Currently, it's difficult to
+set an internal IP.
```

Comment:
Can you add a link to the issue where we are tracking this?

Reply:
Done